### PR TITLE
[mono][tests] Disable CustomGeneratedAttributes test on linux-bionic

### PR DESCRIPTION
**Pull request by kotlarmilos**
## Description

This PR disables failing library test on linux-bionic.

### CODE REVIEW

File: src/libraries/Microsoft.Extensions.Options/tests/SourceGeneration.Unit.Tests/OptionsRuntimeTests.cs [hunk 1]
```diff
@@ -266,7 +266,7 @@ public void TestNewDataAnnotationFailures()
             }, result.Failures);
         }
 #endif // NET8_0_OR_GREATER
-
+        [SkipOnPlatform(TestPlatforms.LinuxBionic, "https://github.com/dotnet/runtime/issues/93723")]
```
Comment (steveisok) [inline src/libraries/Microsoft.Extensions.Options/tests/SourceGeneration.Unit.Tests/OptionsRuntimeTests.cs:269]: For whatever reason, this doesn't seem to be respected on the latest CI run

Comment (kotlarmilos) [reply src/libraries/Microsoft.Extensions.Options/tests/SourceGeneration.Unit.Tests/OptionsRuntimeTests.cs:269]: Correct, here is the latest run: https://dev.azure.com/dnceng-public/public/_build/results?buildId=444522&view=logs&j=8ce12db4-5178-5d5b-6012-cb5f9bb20b0f.

Closing this PR. If the same error is encountered we will reopen it.

**Comment by kotlarmilos** [comment]
Fixed by https://github.com/dotnet/runtime/pull/93709.

**Pull request closed without merging**